change/@fluentui-web-components-eb7910d3-4886-4c46-a3d3-4cb448a176eb.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentui-web-components-v3 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29581&commitId=5862d743eddcc86fa6e4cd1e62074d8e9665d3bd&env=prod&clientType=FLUENTUI)


  <details><summary><b>Switch</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Switch.Checked - RTL.normal.chromium.png|97| Changed|
|Switch.Checked - RTL.hover.chromium.png|104| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>